### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by baharbaba** [community/community#138099] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It's been one week since I submitted my GitHub Support Ticket, but I haven't received any response yet. I would have been more patient, but after seeing some discussions, I've realized that others have waited well over a month for a reply. Is GitHub Support short-staffed, or could there be an issue with the ticketing system?

**Comment by PascalCorpsman** [comment]
Maybe your question is difficult, i got a result in under a day ..

**Reply by baharbaba** [reply]
im waiting like 8 days no response i sent like 2 videos to help solve my problem

**Comment by SomethingNew71** [comment]
I am a paid github user and my account was flagged due to a compromised API key and shadowbanned. I have been waiting a month with no response so I wouldn't hold you breath on a quick response.
